scapy/automaton.py

```diff
@@ -31,11 +31,32 @@
 from scapy.plist import PacketList
 from scapy.data import MTU
 from scapy.supersocket import SuperSocket
+from scapy.packet import Packet
 from scapy.consts import WINDOWS
 import scapy.modules.six as six
 
+from scapy.compat import (
+    Any,
+    Callable,
+    DecoratorCallable,
+    Deque,
+    Dict,
+    Generic,
+    Iterator,
+    List,
+    Optional,
+    Set,
+    Tuple,
+    Type,
+    TypeVar,
+    Union,
+    _Generic_metaclass,
+    cast,
+)
+
 
 def select_objects(inputs, remain):
+    # type: (List[Any], Union[float, int, None]) -> List[Any]
```

Comment:
Is it possible to use a stricter type than Any here?

Reply:
It appears python itself isn't sure how to type this better: https://github.com/python/typeshed/blob/994b69ef8f18e76689daca3947879c3d7f76173e/stdlib/select.pyi#L28

Reply:
too bad =/